capstone/src/test/java/DeleteFlagsTest.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.*;
+import java.util.Date;
+
+public class DeleteFlagsTest {
+    @Test
+    public void testIsTooOld1() throws Throwable {
```

Comment:
name tests in a way that makes it clear what's being tested (get rid of the 1 and add details about why each testIsTooOld is different than the others)

Reply:
Addressed in new pull request.